packages/orchestrator-ui-components/src/components/WfoWorkflowSteps/WfoWorkflowStepList/WfoWorkflowStepList.tsx

```diff
@@ -90,20 +124,24 @@ export const WfoWorkflowStepList = React.forwardRef(
                 <WfoStepListHeader
                     showHiddenKeys={showHiddenKeys}
                     showRaw={showRaw}
+                    showDelta={showDelta}
                     allDetailToggleText={
                         allStepsAreExpanded ? t('collapseAll') : t('expandAll')
                     }
                     onChangeShowHiddenKeys={setShowHiddenKeys}
                     onChangeShowRaw={setShowRaw}
+                    onChangeShowDelta={setShowDelta}
                     onToggleAllDetailsIsOpen={() =>
                         setExpandedStateStepListItems(!allStepsAreExpanded)
                     }
                     isTask={isTask}
                 />
 
-                {showRaw ? (
-                    <WfoProcessRawData processId={processId} />
-                ) : (
+                {showRaw && <WfoProcessRawData processId={processId} />}
+                {showDelta && (
```

Comment:
is there a state possible where Raw and Delta are both shown? Maybe consider introducing an enum with values:
- raw
- delta
- steplist

Reply:
for now, I am happy with this; if more state changes are needed; I would consider splitting in to separate components.